### PR TITLE
docs: add zh-CN translations

### PR DESCRIPTION
**Pull request by Jeff-Tian**
## Description

Hello, there! I love this great library and add zh-CN translations for it. Is it possible to get it merged? The deployed preview can be found here: https://inversifyjs.netlify.app/zh_cn/

## Motivation and Context

Add zh-CN translations to motivate more Chinese developers to use InversifyJS.

## How Has This Been Tested?

Only documentation changes, deployed preview here: https://inversifyjs.netlify.app/zh_cn/

## Types of changes

- [x] Updated docs / Refactor code / Added a tests case (non-breaking change)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [ ] I have updated the changelog.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -19,4 +19,4 @@
     "tslint.autoFixOnSave": true,
     "typescript.tsdk": "./node_modules/typescript/lib",
     "vsicons.presets.angular": false
-}
```
Comment (dcavanagh) [inline .vscode/settings.json]: revert this please

File: wiki/en/README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Wiki
 
-Welcome to the InversifyJS wiki!
```
Comment (dcavanagh) [inline wiki/en/README.md]: Revert this white space change please

File: netlify.toml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# example netlify.toml
```
Comment (dcavanagh) [inline netlify.toml:1]: What does netify do? Does this file need to be committed?

File: wiki/en/SUMMARY.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
+# Wiki
```
Comment (dcavanagh) [inline wiki/en/SUMMARY.md:1]: Should this file be named README.md? It duplicates README.md and adds the translation

File: wiki/en/glossary.md [hunk 5]
```diff
@@ -1,4 +1,5 @@
 # Glossary for users
+
```
Comment (dcavanagh) [inline wiki/en/glossary.md]: Remove white space changes

**Review by dcavanagh — CHANGES_REQUESTED** [review]
Thank you for this. It looks like it was a lot of work. I made a few comments. I cannot review the CN language portions. We should have someone review them to back you up.

File: netlify.toml (hunk 3 above)
Comment (Jeff-Tian) [reply netlify.toml:1]: Netlify is a static site host with CDN enabled. Highly recommend using it as github wiki is slow and can be error when accessing from China.

Comment (Jeff-Tian) [reply netlify.toml:1]: I've tested the deploying to netlify works: https://doc.inversify.cloud/zh_cn/

File: wiki/en/SUMMARY.md (hunk 4 above)
Comment (Jeff-Tian) [reply wiki/en/SUMMARY.md:1]: Renaming it to SUMMARY.md to match the gitbook style. The gitbook is static site generator.

**Comment by dcavanagh** [comment]
@Jeff-Tian  @NeoYo Are you able to have someone review the translation to back you up?

**Comment by Jeff-Tian** [comment]
@dcavanagh thanks for your comments!



> @Jeff-Tian @NeoYo Are you able to have someone review the translation to back you up?

Not yet, but it's not urgent. Maybe just wait and let's see who can help in the future.

**Comment by jkhmtx** [comment]
@Jeff-Tian @dcavanagh for a lack of another Chinese speaker to review this, I suggest that we give the benefit of the doubt and allow this to be merged after updating. What do you think?

**Comment by Jeff-Tian** [comment]
Sure. Closing for now.

**Pull request closed without merging**